src/constants.py

```diff
@@ -29,7 +29,7 @@
 # Snap constants.
 PGBACKREST_EXECUTABLE = "charmed-postgresql.pgbackrest"
 POSTGRESQL_SNAP_NAME = "charmed-postgresql"
-SNAP_PACKAGES = [(POSTGRESQL_SNAP_NAME, "14/edge")]
+SNAP_PACKAGES = [(POSTGRESQL_SNAP_NAME, {"revision": 31})]
```

Comment:
If we don't intend to install snaps based on channel, we can just put the revision here.